grammar.js

```diff
@@ -92,7 +92,8 @@ module.exports = grammar({
     // manually controlled by adding the following to parenthesized rules
     _newline: (_$) => /\r?\n/,
     _terminator: (_$) => choice(PUNC().semicolon, _$._newline),
-    _separator: (_$) => choice(/[ \t]+/, _$._newline),
+    _space: (_$) => /[ \t]+/,
+    _separator: (_$) => choice(_$._space, _$._newline),
```

Comment:
Woudn't `_$` indicate that the parameter is unused? `_terminator` seems to violate this assumption as well. What is your reasoning behind it?

Reply:
I thought the naming of `_$` means terminal token

Reply:
@mrdgo you're right, referencing to other named rules makes them non-terminal then. I'll change the naming

Reply:
Well, then we're both right, aren't we? :D

Reply:
Yeah, good catch